MatomoTracker.php

```diff
@@ -1297,16 +1292,11 @@ protected function loadVisitorIdCookie()
         if (strlen($parts[0]) != self::LENGTH_VISITOR_ID) {
             return false;
         }
+
         /* $this->cookieVisitorId provides backward compatibility since getVisitorId()
-        didn't change any existing VisitorId value */
+didn't change any existing VisitorId value */
         $this->cookieVisitorId = $parts[0];
         $this->createTs = $parts[1];
-        $this->visitCount = (int)$parts[2];
```

Comment:
@diosmosis we will still also need to support the old cookies just in case. That's because say

* You update Matomo to Matomo 4
* You also update the PHP tracker
* Then a visitor visits your website who didn't visit your website in a while. Then the old cookie structure will be sent. Possible that not even the tracking code is embedded on that same page.

We could check maybe how many parts are included and then use old or new strategy?

Reply:
@tsteur The only parts of the old cookie that are used in Matomo 4 are the first two elements. So if a cookie w/ more items comes in, it should still work. Unless I'm missing something? (There's a test for this, btw.)